force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -94,16 +110,22 @@ public with sharing class DummyJSONCallout {
 	
 	public static void postCreateDummyJSONUser(String contactId) {
 		// Create HTTP request to send.
-
+		HttpRequest request = new HttpRequest();
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
+		request.setEndpoint('callout:DummyJsonUser');
 		// Set the HTTP method to POST.
-
+		request.setMethod('POST');
 		// Set the body using generateDummyJsonUserPayload method.
-
+		request.setBody(generateDummyJsonUserPayload(contactId));
 		// Send the HTTP request and get the response.
-
+		Http http = new Http();
+		HTTPResponse response = http.send(request);
 		// If the HTTP response code is successful, update the contact.
+		if(response.getStatusCode() >= 200 || response.getStatusCode() <= 299) {
+			// Update the contact's DummyJSON_Last_Updated__c field to the current time.
+			Contact cont = new Contact(Id = contactId, DummyJSON_Last_Updated__c = Datetime.now());
```

Comment:
Try to do a check to see if the contactId that was passed in was null, if null throw an IllegalArgumentException